extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/devmode/tests/TestStatus.java

```diff
@@ -80,6 +91,33 @@ public TestStatus setTestsSkipped(long testsSkipped) {
         return this;
     }
 
+    public long getTotalTestsPassed() {
+        return totalTestsPassed;
+    }
+
```

Comment:
Silly question, but why having setters here if the existing constructor already initializes the state?

Reply:
Its serialized to JSON.

Reply:
That's fine then. I know it can be bypasssed using `@JsonCreator` but that would introduce a hard dep on Jackson